p2pd/main.go

```diff
@@ -251,6 +257,10 @@ func main() {
 		}
 	}
 
+	c.Security.SECIO = *useSecio
+	c.Security.TLS = *useTls
+	c.Security.Noise = *useNoise
```

Comment:
Won't this override the config if one was passed as an argument?
You might need to gate the options.

Reply:
To clarify, we do want to override the config -- but only if the option was passed in the command line.
As it is right now, it unconditionally overrdes.